app/Http/Controllers/CheckoutController.php

```diff
@@ -13,6 +13,13 @@
 class CheckoutController extends Controller
 {
      public function checkout(Request $request){
+      $selectedOfferId = $request->selectedOfferId;
+
+      #select the payment period and percentag from the lender offering table
+      $lenderOffering = DB::table('lender_offerings')->select('payment_period','percentage','max_financed','lender_organization_id')->where('id',$selectedOfferId)->first();
+
+      //TODO :: check if the total price is greater than the max financed amount from the table 
```

Comment:
You will need the total product price to do this, as it is found it the view file, you can place a simple check there inside the for loot that displays the list of offers 
ie. If `product_price  +  commission is greater than max financed don't display `it in the list